src/api/java/com/minecolonies/api/entity/pathfinding/SurfaceType.java

```diff
@@ -128,6 +136,25 @@ public static SurfaceType getSurfaceType(final BlockGetter world, final BlockSta
         return SurfaceType.DROPABLE;
     }
 
+    /**
+     * Is the surface inherently dangerous to stand on/in (i.e. causes damage).
+     *
+     * @param blockState block to check.
+     * @return           true if dangerous.
+     */
+    public static boolean isDangerous(final BlockState blockState)
+    {
+        final Block block = blockState.getBlock();
+
+        return  blockState.is(ModTags.dangerousBlocks) ||
+                block instanceof FireBlock ||
+                block instanceof CampfireBlock ||
+                block instanceof AbstractBlockBarrel ||
```

Comment:
dont need the barrel as dangerous block anymore, that was an old legacy check/leftover